Makefile

```diff
@@ -1,9 +1,13 @@
 init: Pipfile
+	pip install pipenv --user
+	pipenv install --python 3
```

Comment:
Installing stuff with pip is bad idea, even with --user,
but if we want to use it a better way is:

    python3 -m pip ...

This way it works correctly for the used python without extra work.

Why not `dnf install pipenv`?